reactor-extra/src/test/java/reactor/math/ReactorMathTests.java

```diff
@@ -177,6 +177,22 @@ public void emptySumBigInteger() {
 		verifyEmptyResult(MathFlux.sumBigInteger(Mono.empty()));
 	}
 
+	@Test
+	public void monoBigIntegerRange() {
```

Comment:
rename to `noOverflowSumBigInteger` and split out the `averageBigInteger` case into `noOverflowAverageBigInteger()` (to be placed under the `emptyAverageBigInteger` one)